src/main.cpp

```diff
@@ -2176,6 +2218,11 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
         flags |= SCRIPT_VERIFY_CHECKLOCKTIMEVERIFY;
     }
 
+    // Start enforcing CHECKSEQUENCEVERIFY using versionbits logic.
```

Comment:
I don't have a strong opinion here, but I don't know that it really makes sense to call a generic `GetFlags()` function, as I believe we will need to have a separate one of those to deploy BIP68 (which doesn't use a script flags variable, but uses a locktimeflags variable instead)